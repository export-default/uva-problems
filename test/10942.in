52
1073741823 99 1 2
3 0 2 29
41 5 11 68
19169 7 9 25
26962 6 2 65
23281 12 11 28
2995 1 23 43
32391 10 28 5
292 9 31 83
19718 7 23 96
14771 5 10 39
25667 7 17 100
28703 10 13 12
17673 8 14 65
28253 9 4 69
32662 8 12 58
8723 11 2 42
12316 14 3 36
288 15 5 7
19264 29 3 49
15890 6 11 30
15006 15 10 2
19629 22 1 24
18756 30 11 41
13931 14 1 9
24626 25 6 24
16118 19 10 83
4833 23 8 16
22704 13 10 31
31673 9 6 87
26924 2 7 73
26777 21 10 74
23986 37 17 6
22355 75 3 11
4031 51 9 4
16941 31 17 4
31107 38 28 11
15457 84 9 11
14945 59 1 5
24221 47 6 0
27506 69 15 10
900 56 8 11
17410 38 4 11
21548 42 6 3
3602 37 31 2
9374 22 9 4
27348 85 15 3
8281 100 23 6
26418 89 19 2
18127 94 9 11
24648 22 14 7
14310 15 29 5
