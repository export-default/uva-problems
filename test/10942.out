03 06 09
-1
